### PR TITLE
Allow gatk java option GATK_STACKTRACE_ON_USER_EXCEPTION to be passed onto the native library

### PR DESCRIPTION
**Pull request by nalinigans**

### CODE REVIEW

File: tests/run.py [hunk 1]
```diff
@@ -235,6 +235,29 @@ def substitute_workspace_dir(jsonfile, wsdir):
...
+    if(pid.returncode != 0):
+        sys.stderr.write('Query with java options : '+query_command+' failed\n');
+        cleanup_and_exit(tmpdir, -1);
+    actual_md5sum = str(hashlib.md5(stdout_string).hexdigest())
+    golden_str, golden_md5sum = get_file_content_and_md5sum('golden_outputs/java_t0_1_2_vcf_at_0')
+    if (actual_md5sum != golden_md5sum):
+        sys.stderr.write('Query with java options : '+query_command+' failed. Results did not match with golden output.\n');
+        cleanup_and_exit(tmpdir, -1)
+    sys.stdout.write("Successful\n")
+
```
Comment (mlathara) [inline tests/run.py:260]: Question about this test...is it supposed to be testing the stack trace option is being passed in correctly and that it gets printed out? Or something else?

Comment (nalinigans) [reply tests/run.py:260]: It is testing only the JNI code for passing down the stack trace option more like a unit test really. I was looking at adding this to the junits, but there seems to be no clean way to get multiple class loaders in the tests. The query itself runs cleanly, so there is not exception to track.

Comment (mlathara) [reply tests/run.py:260]: Gotcha - a little nitpicky, but I wonder if it is worth it to manipulate the jsons (or the workspace) to get it to throw an exception and check that?

**Review by mlathara — DISMISSED** [review]

File: tests/run.py (hunk 1 above)
Comment (nalinigans) [reply tests/run.py:271]: Done.

**Review by mlathara — APPROVED** [review]
looks good

**Pull request merged**